recipes/iqtree/meta.yaml

```diff
@@ -19,15 +19,18 @@ requirements:
   host:
     - eigen
     - zlib
+    - openmp # [osx]
   run:
     - zlib
+    - openmp # [osx]
 
 test:
   commands:
-    - iqtree-omp
```

Comment:
Was this dropped from the tool?

Reply:
Yes. Since version 1.6.0, the two versions of IQ-TREE were merged.

http://www.iqtree.org/release/